puppetboard/templates/reports.html

```diff
@@ -27,7 +27,7 @@
     {% endfor %}
```

Comment:
Nit: I know that it's out of the scope of the PR, but it would be nice to unify this to look and work the same way as such filter in the `/nodes` page.

Reply:
Yes i will do that later. When this PR is ok / merged, i will update my other one. After that i will open a new one for UI unification and improvements.  

To be honest, in the long run it would be easier to add a real frontend and only manage json returns on the backend. Remove all JS / CSS dependencies and start from scratch with, for example, the vuejs library and the element-plus framework.

Frontend update management would be simplified and optimized to handle large amounts of data

Reply:
I totally agree that the frontend of the app just needs to be replaced with some modern, standard solution. I don't have the skills to do that though, but if you are willing to implement it then it's great and you are welcome to do it! I will do my best to help you test it, review the code, get it merged and released. :)

Reply:
I 100% agree with @gdubicki . I do not have the skills to write the frontend, but I think its a really good idea. I am happy to review and test PRs.

Reply:
Perhaps we should sync on Voxpupuli slack regarding this, @melck? If you want, please join it at https://puppetcommunity.slack.com/messages/voxpupuli/ . @bastelfreak and me are there under the same ids as here.